VotingApplication/VotingApplication.Web.Api.Tests/E2E/Helpers/Clearers/DBClearer.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Data.Entity;
+using VotingApplication.Data.Context;
+using VotingApplication.Data.Model;
+
+namespace VotingApplication.Web.Api.Tests.E2E.Helpers.Clearers
+{
+    public class DBClearer : Clearer
+    {
+        public DBClearer(IVotingContext context) : base(context) { }
+
+        public void ClearAll()
```

Comment:
From what I can tell, this isn't actually called anywhere? (Despite it being the most appropriate for the tests, I'd argue).